src/test/java/com/giftmaseya/conveyorservice/service/impl/CalculationServiceImplTest.java

```diff
@@ -0,0 +1,210 @@
+package com.giftmaseya.conveyorservice.service.impl;
+
+import com.giftmaseya.conveyorservice.dto.EmploymentDTO;
+import com.giftmaseya.conveyorservice.dto.ScoringDataDTO;
+import com.giftmaseya.conveyorservice.exception.ConveyorException;
+import com.giftmaseya.conveyorservice.utils.EmploymentStatusEnum;
+import com.giftmaseya.conveyorservice.utils.GenderEnum;
+import com.giftmaseya.conveyorservice.utils.MaritalStatusEnum;
+import com.giftmaseya.conveyorservice.utils.PositionEnum;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+
+import static com.giftmaseya.conveyorservice.utils.EmploymentStatusEnum.BUSINESS_OWNER;
+import static com.giftmaseya.conveyorservice.utils.PositionEnum.MIDDLE_MANAGER;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+
+class CalculationServiceImplTest {
```

Comment:
It's better to use junit5 and Mocks frameworks for testing. See our lecture about testing to know how.